ci/run_ci.sh

```diff
@@ -0,0 +1,105 @@
+
+#!/bin/bash
+
+#set -x
+
+
+#smoke test environment specifications:
+#for scale perf
+export SCALE=6
+export RUNS=1
+export TIMEOUT=10
+#for kube burner 
+export JOB_ITERATIONS=3
+export NODE_COUNT=2
+export PODS_PER_NODE=2
+#for upgrade perf
+export VERSION=4.6.12
+#for router perf
+export PBENCH_SERVER='pbench.dev.openshift.com'
+export COMPARE=false
+export HTTP_TEST_SUFFIX='smoke-test'
+export HTTP_TEST_SMOKE_TEST=true
+
+
+#pull requests list
+git ls-remote origin 'pull/*/head' | awk '{print $2}' |
+while read ref; do
+  pr=$(echo $ref | cut -d/ -f3)
+  git fetch origin $ref > /dev/null
+  files_changed=$(git show --pretty=format:'' --name-only FETCH_HEAD|wc -l)
+  echo "PR number $pr has changes in $files_changed files"
+done
+
+
+#git checkout /pr/<pr-no>   the pr number can be accepted as input or obtained from pr_list
+
+diff_list=`git diff --name-only origin/master`
+
+
+if [ -z "$diff_list" ]
+then
+      echo "No changes noticed , exiting"
+      exit 0
+else
+      echo -e " List of files changed : ${diff_list} "
+fi
+
+cat > results.markdown << EOF
+Results for e2e-benchmarking CI Test
+Test                                             | Result | Runtime  |
+-------------------------------------------------|--------|----------|
+EOF
+
+test_list=`find ./workloads/ -maxdepth 2 -name '*t.sh' -type f `
```

Comment:
This might create some issues in the future, in case we have some file name not following "*t.sh" pattern.
Instead, create a run_$Workload_ci.sh file in each workload which has command to run the required scripts and call them here.